src/main/java/control/FileMenuController.java

```diff
@@ -0,0 +1,150 @@
+package control;
+
+import java.io.File;
+
+import data.CameraShot;
+import data.CameraTimeline;
+import data.DirectorTimeline;
+import data.ScriptingProject;
+import gui.centerarea.CameraShotBlock;
+import gui.events.NewProjectCreationEvent;
+import gui.modal.NewProjectModalView;
+import gui.root.RootCenterArea;
+import gui.root.RootPane;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Alert.AlertType;
+import javafx.scene.input.MouseEvent;
+import javafx.stage.FileChooser;
+import javafx.stage.FileChooser.ExtensionFilter;
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+public class FileMenuController {
+    
+    
+    private ControllerManager controllerManager;
+    
+    public FileMenuController(ControllerManager manager) {
+        this.controllerManager = manager;
+    }
+    
+    /**
+     * Save the current project state to file.
+     * A file chooser window will be opened to select a file
+     */
+    public void save() {
+        log.info("Saving Project to file");
+        FileChooser fileChooser = new FileChooser();
+        fileChooser.setTitle("Save");
+        ExtensionFilter extFilter = new ExtensionFilter("txt files", "*.txt");
+        fileChooser.getExtensionFilters().add(extFilter);
+        File file = fileChooser.showSaveDialog(controllerManager.getRootPane().getPrimaryStage());
+        if (file != null) {
+            controllerManager.getScriptingProject().write(file);
+        }
+    }
+    
+    /**
+     * Load a project from file.
+     * A file chooser window will be opened to select the file.
+     */
+    public void load() {
+        log.info("Loading Project from file");
+        FileChooser fileChooser = new FileChooser();
+        fileChooser.setTitle("Load");
+        ExtensionFilter extFilter = new ExtensionFilter("txt files", "*.txt");
+        fileChooser.getExtensionFilters().add(extFilter);
+        File file = fileChooser.showOpenDialog(controllerManager.getRootPane().getPrimaryStage());
+        if (file != null) {
```

Comment:
Again, it means the user did not select a file, so the user should not get any extra error messages because he knows he aborted the file selection himself